src/main/java/com/zendesk/maxwell/row/RowMap.java

```diff
@@ -230,8 +230,9 @@ public String toJSON(MaxwellOutputConfig outputConfig) throws IOException {
 		}
 
 		if ( outputConfig.includesBinlogPosition )
-			g.writeStringField("position", this.nextPosition.getFile() + ":" + this.nextPosition.getOffset());
-
+			g.writeStringField("position",
+				(nextPosition.getGtidStr() != null ? nextPosition.getGtidStr() + ":" : "")
+				+ this.nextPosition.getFile() + ":" + this.nextPosition.getOffset());
```

Comment:
probably either the gtid here *or* the binlog position?   Better yet, move gtid into its own field `gtid_position` or something.